config/monitoring/metrics/grafana/100-grafana-dash-eventing.yaml

```diff
@@ -17,6 +17,608 @@
 # --patch "$(cat 100-grafana-dash-eventing.yaml)" -n knative-monitoring
 ---
 data:
+  knative-eventing-reconciler-dashboard.json: |+
```

Comment:
The eventing controllers will show up in the `Knative Serving Controller` dashboards from my testing.

Reply:
True that. I removed this dashboard completely. We'll use `Knative Serving - Reconciler` dashboard but I'll change the name to `Knative - Reconciler`(more generic). And of course, we'll take all of the monitoring components out of serving as in [here](https://github.com/knative/serving/issues/5254). By the way, where can I find `Knative Serving Controller` dashboard? I just see the following dashboards.
<img width="713" alt="Screen Shot 2019-08-23 at 15 18 24" src="https://user-images.githubusercontent.com/5954057/63595499-6542a980-c5b9-11e9-9267-0a4f9b5994fc.png">